### PR TITLE
Update Urbit to v2.0

### PR DESCRIPTION
**Pull request by yapishu**
- Updated Urbit to official v2.0 binary: https://github.com/urbit/vere/releases

### CODE REVIEW

**Comment by yapishu** [comment]
Added `PROXY_AUTH_ADD: "false"` to compose by request: https://github.com/mopfel-winrux/urbit-umbrel/issues/24

File: urbit/docker-compose.yml [hunk 1]
```diff
@@ -4,8 +4,9 @@ services:
     environment:
       APP_HOST: urbit_manager_1
       APP_PORT: 8090
+    PROXY_AUTH_ADD: "false"
   
-  manager:
+    image: mopfelwinrux/urbit-umbrel:v2.0@sha256:5c628df5007e3c3e8320b6e3f2e97a120e5a448ab5821bbe5fb21c922d64d3e4
     image: mopfelwinrux/urbit-umbrel:v1.22@sha256:fd6303520a4ef3dec9be223b099af6a2299810a277b921e574e135a96a2eb39f
```
Comment (nmfretz) [inline urbit/docker-compose.yml:10]: The `manager` service heading was accidentally deleted and now there are two images (new v2.0 and v.1.22).

File: urbit/docker-compose.yml [hunk 2]
```diff
@@ -4,8 +4,9 @@ services:
     environment:
       APP_HOST: urbit_manager_1
       APP_PORT: 8090
+    PROXY_AUTH_ADD: "false"
```
Comment (nmfretz) [inline urbit/docker-compose.yml:7]: sounds good. Your app is still protected by its own auth.

**Review by nmfretz — COMMENTED** [review]
I checked the new v2.0 image. This will be good to go with the correction noted below.

File: urbit/docker-compose.yml (hunk 1 above)
Comment (yapishu) [reply urbit/docker-compose.yml]: Woops! Adding the new env var line broke the parser in my update script and I missed this. Fixed now

**Comment by nmfretz** [comment]
Tested on amd64 and arm64. Going live now. Good luck with the v2.0 launch!

**Pull request merged**